dali/operators/reader/loader/numpy_loader.cc

```diff
@@ -244,13 +73,13 @@ void NumpyLoader::ReadSample(NumpyFileWrapper& target) {
   auto current_file = FileStream::Open(path, read_ahead_, !copy_read_data_);
 
   // read the header
-  NumpyHeaderMeta header;
+  numpy::HeaderData header;
   auto ret = header_cache_.GetFromCache(filename, header);
   try {
     if (ret) {
       current_file->SeekRead(header.data_offset);
     } else {
-      detail::ParseHeader(current_file.get(), header);
+      header = numpy::ParseHeader(current_file.get());
```

Comment:
In this case, this assignment is counter-productive.
Also, I think ADL should make it work without a namespace.
```suggestion
      ParseHeader(header, current_file.get());
```

Reply:
Ok, I'll change that in a moment